### ISSUE TITLE
Scoped Organization Private NPM packages

### ISSUE THREAD

**Issue by j** [lerna/lerna#1735]
All my packages are marked as private and I'm expecting them to be able to be published to NPM as private NPM modules within our organization.

Is this something lerna supports?

**Comment by busticated** [comment]
yep, works great. you need to use the `publishConfig` field in your `package.json` file ([docs](https://docs.npmjs.com/files/package.json#publishconfig)) instead of simply `private: true`

here's what i use:

```json
"publishConfig": {
    "access": "restricted"
}
```

**Comment by j** [comment]
Yeah, I got it to work actually, it's just not very intuitive.  I'd expect a "private: true" package to always be private on NPM no matter what, so just specifying that it's publishable should publish as private.

I'd like to see lerna have a global publish config that makes every package private by default unless specified otherwise.  That way someone won't accidentally publish private stuff.
